src/lib/notifications/backgroundFetch.native.js

```diff
@@ -144,7 +155,6 @@ export const initBGFetch = once((goodWallet, userStorage) => {
   const hasConnection = async () => Promise.race([Promise.all([goodWallet.ready, userStorage.ready]), waitUntil(10000)])
 
   BackgroundFetch.configure(options, task, taskManagerErrorHandler)
-  BackgroundFetch.registerHeadlessTask(androidHeadlessTask)
   BackgroundFetch.scheduleTask({ taskId: FEED_NOTIFICATIONS })
   BackgroundFetch.scheduleTask({ taskId: CLAIM_NOTIFICATION })
```

Comment:
this is triggers a "one time" task for all i know.

Reply:
No if we set "periodic: true" for bg-fetch configuration

Reply:
@alt-a1r 
according to their doc
```
periodic
Valid only for BackgroundFetch.scheduleTask. Defaults to false. Set true to execute the task repeatedly. When false, the task will execute just once.
```
see in their docs how they use it
`scheduleTask({taskId:x, periodic: true})`